src/Form/InputCheckbox.php

```
@@ -0,0 +1,227 @@
+    {
+        $attributes = parent::getAttributes();
+
+        /** @var mixed $value */
+        $value = $attributes['value'] ?? null;
+
+        if (is_bool($value)) {
+            $value = $value ? 1 : 0;
+            $attributes['value'] = $value;
+        }
+
+        $normalizedChecked = Enum::normalizeValue($this->checked);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'getAttributes'. (undefined)

(StaticAccess)